### PR TITLE
Fixed Issue 2747 tapping on thanks notification takes to thanks logs

### LINKED ISSUE #2747: Clicking on Sending thanks should take to thank logs
**Summary:** 

**Expected**
After "Send Thanks" notification is completed clicking on notification should take us to "Thanks Log" for the image.  

**Observed**
Currently it opens Home page of the Application

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by vanshikaarora**
**Description (required)**

Tapping on "Successfully sent thanks to ......" will take to Thanks Log activity.

Fixes #2747 Clicking on Sending thanks should take to thank logs

### Changes made
Added Pending Intent for thanks log url

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/review/ReviewController.java [hunk 1]
```diff
@@ -38,6 +42,7 @@
     private NotificationManager notificationManager;
     private NotificationCompat.Builder notificationBuilder;
     private Media media;
+    private static final String THANKS_LOG="https://en.wikipedia.org/wiki/Special:Log/thanks";
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/review/ReviewController.java]: It should go to https://commons.wikimedia.org/wiki/Special:Log/thanks or https://commons.wikimedia.beta.wmflabs.org/wiki/Special:Log/thanks instead, depending to the beta flavor.

File: app/src/main/java/fr/free/nrw/commons/review/ReviewController.java [hunk 2]
```diff
@@ -209,6 +214,9 @@ public void sendThanks(@NonNull Activity activity) {
                             .setProgress(0, 0, false)
                             .setOngoing(false)
                             .setPriority(NotificationCompat.PRIORITY_HIGH);
+                    Intent browserIntent = new Intent(Intent.ACTION_VIEW , Uri.parse(THANKS_LOG));
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/review/ReviewController.java]: Please add some comments before this line.

File: app/src/main/java/fr/free/nrw/commons/review/ReviewController.java [hunk 3]
```diff
@@ -209,6 +215,16 @@ public void sendThanks(@NonNull Activity activity) {
...
                             .setOngoing(false)
                             .setPriority(NotificationCompat.PRIORITY_HIGH);
+                    /**
+                     * On tapping the notification it should take to visit the Thanks Log page
+                     * Thus adding a pending content to redirect to a url depending on the beta flavor*/
+                    String url;
+                    if (ConfigUtils.isBetaFlavour())
+                        url = BuildConfig.HOME_URL+"Special:Log/thanks";
+                    else url = THANKS_LOG;
+                    Intent browserIntent = new Intent(Intent.ACTION_VIEW , Uri.parse(url));
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/review/ReviewController.java]: We generally use If and elses with braces, please add them too:)

Comment (neslihanturan) [reply app/src/main/java/fr/free/nrw/commons/review/ReviewController.java]: Actually this if/else can be removed at all. You can just use BuildConfig.Home_URL variable. Is returns the correct URL according to flavour.

Comment (vanshikaarora) [reply app/src/main/java/fr/free/nrw/commons/review/ReviewController.java]: @neslihanturan I have made the changes :+1:

**Comment by vanshikaarora** [comment]
This PR has been pending since long. I have resolved the merge conflicts. @neslihanturan now it's up for review

**Comment by neslihanturan** [comment]
Hi @vanshikaarora , I approved it by reading the code. But on my tests, clicking to text does not brings me to anywhere, both for beta and prod.

**Comment by madhurgupta10** [comment]
@vanshikaarora Let us know if you are still working on this.

**Comment by neslihanturan** [comment]
Seems like this one is sleeping, closing for now. Please feel free to get help from here to solve this issue in future. Please feel free to reopen this one if you feel like interested @vanshikaarora :)

**Pull request closed without merging**